atomsci/ddm/pipeline/diversity_plots.py

```diff
@@ -396,15 +395,15 @@ def _obach_diversity_plots(ecfp_radius=6):
     cmpd_df = pd.read_csv(cmpd_file, index_col=False)
     compound_ids = cmpd_df[id_col].values
     smiles_strs = cmpd_df[smiles_col].values
-    ncmpds = len(smiles_strs)
+    _ncmpds = len(smiles_strs)
```

Comment:
delete this function and other dataset-specific ones

Reply:
Deleted https://github.com/ATOMScience-org/AMPL/commit/6227792ae02fc2b634b6fe45992f2f8b27d44999